src/identity.js

```diff
@@ -1899,4 +1803,62 @@ export default function Identity(mpInstance) {
         }
         return userAttributeChangeEvent;
     };
+
+    this.reinitForwardersOnUserChange = function(prevUser, newUser) {
+        if (didUserChange(prevUser, newUser)) {
+            mpInstance._Forwarders.initForwarders(
+                newUser.getUserIdentities().userIdentities,
+                mpInstance._APIClient.prepareForwardingStats
+            );
+        }
+    };
+
+    this.setForwarderCallbacks = function(user, method) {
+        // https://go.mparticle.com/work/SQDSDKS-6036
+        mpInstance._Forwarders.setForwarderUserIdentities(
+            user.getUserIdentities().userIdentities
+        );
+        mpInstance._Forwarders.setForwarderOnIdentityComplete(user, method);
+        mpInstance._Forwarders.setForwarderOnUserIdentified(user);
+    };
+}
+
+function didUserChange(prevUser, newUser) {
```

Comment:
let's export and test this function

Reply:
I'd also be open to moving a bunch of these newly created functions to a TS module, then importing them here to put us in a better position for next gen.